### PR TITLE
test(react): icTextField Cypress Tests

### PR DESCRIPTION
**Pull request by MI6-286**
icTextField Cypress Tests

#1768

## Summary of the changes
Completed migration of e2e tests to Cypress for IcTextField and removed old e2e tests.

## Related issue
#1768

### CODE REVIEW

File: packages/react/src/component-tests/IcTextField/IcTextFieldTestData.tsx [hunk 1]
```diff
@@ -31,3 +34,362 @@ export const Uncontrolled = () => {
...
+  (): ReactElement => (
+    <div style={style}>
+      <IcTextField
+        maxLength={25}
+        label="What is your favourite coffee?"
+        required
+        placeholder="Please enter…"
+        helperText="Such as Arabica, Robusta or Liberica"
+      >
+        <SlottedSVG
```
Comment (GCHQ-Developer-299) [inline packages/react/src/component-tests/IcTextField/IcTextFieldTestData.tsx:54]: Not at all necessary for approval on this PR, but I think you can extract the SlottedSVG into it's own variable and paste than it for each example in the file, to cut down on duplicated markup :)

Comment (GCHQ-Developer-299) [reply packages/react/src/component-tests/IcTextField/IcTextFieldTestData.tsx:54]: Check out an example in IcSideNavigationTestData - the variable ReusableSlottedIcon

File: packages/react/src/component-tests/IcTextField/IcTextFieldTestData.tsx [hunk 2]
```diff
@@ -31,3 +47,290 @@ export const Uncontrolled = () => {
...
+      helper-text="Such as Arabica, Robusta or Liberica"
+    >
+      <SlottedSVG
+        slot="icon"
+        xmlns="http://www.w3.org/2000/svg"
+        height="24px"
+        viewBox="0 0 24 24"
+        width="24px"
+        fill="#000000"
+        size="small"
```
Comment (ad9242) [inline packages/react/src/component-tests/IcTextField/IcTextFieldTestData.tsx:330]: does this `size` on slottedsvg do anything? i don't think there is a "size" attribute. can this be removed & then just use the ReusableSlottedIcon?

**Review by ad9242 — CHANGES_REQUESTED** [review]

File: packages/react/src/component-tests/IcTextField/IcTextFieldTestData.tsx (hunk 2 above)
Comment (MI6-286) [reply packages/react/src/component-tests/IcTextField/IcTextFieldTestData.tsx]: As agreed, removed the icon from CustomSizeTextField

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Review by ad9242 — APPROVED** [review]

**Pull request merged**